api/src/main/java/org/openmrs/module/fhir2/api/FhirTaskService.java

```diff
@@ -21,15 +21,6 @@
  */
 public interface FhirTaskService extends FhirService<Task> {
 	
-	/**
-	 * Get list of tasks that reference the object type/UUID combo provided
-	 *
-	 * @param basedOnReference
-	 * @param ownerReference
-	 * @param status
-	 * @param sort
-	 * @return the collection of Tasks that match the search parameters
-	 */
```

Comment:
This needs to be updated, not removed.

Reply:
Are we planning to have java docs on both the interfaces and the implementation? I think just having them on the interfaces suffices.

Reply:
Largely just on interfaces, but if FM2-268 is implemented (or as we gradually migrate towards that) it may be necessary to have the Javadoc on some implementations when there is no appropriate interface anymore.